ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op.cpp

```diff
@@ -54,7 +54,7 @@ Tensor optimized_conv_new(const Tensor& a, const Tensor &b, std::optional<const
     bool untilize_out, bool fuse_relu, MathFidelity math_fidelity,
     const OptimizedConvParallelizationConfig& parallelization_config,
     const OptimizedConvBlockConfig& block_config,
-    MemoryConfig memory_config,
+    MemoryConfig out_memory_config,
```

Comment:
do we have to do this rename, seems to add noise to the PR

Reply:
It would be better to be explicit about which tensor the memory config refers to. In the future, I would like to do something similar for block sharding along the width dimension.

Reply:
I think we should leave it as `memory_config`, to be consistent with rest of the TTNN ops (we used to call it various things incl. `out_memory_config`, but then it was changed to `memory_config` in all ops)